tools/executable/utils.go

```diff
@@ -0,0 +1,56 @@
+package executable
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/configwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/gethwrappers"
+)
+
+type ContractDeployBackend interface {
+	bind.ContractBackend
+	bind.DeployBackend
+}
+
+type Operation struct {
+	To    common.Address
+	Data  string
+	Value uint64
+}
+
+type ChainOperation struct {
+	ChainIdentifier string
+	Operation
+}
```

Comment:
Should we have json annotation on this struct like on `ExecutableMCMSProposal`

```suggestion
type Operation struct {
	To    common.Address `json:"to"`
	Data  string `json:"data"`
	Value uint64 `json:"value"`
}

type ChainOperation struct {
	ChainIdentifier string `json:"chainId"`
	Operation
}
```

Reply:
Executable should in theory never be directly written/read from JSON, so I lean no, but it also doesn't hurt.